src/leveragers/Leverager.sol

```diff
@@ -18,35 +17,29 @@ abstract contract Leverager is ILeverager, Ownable {
     address public yieldToken;
     address public underlyingToken;
     address public debtToken;
-    address public flashLoan;
-    address public debtSource;
-    address public dex;
-    IAlchemistV2 alchemist;
+    IAlchemistV2 public alchemist = IAlchemistV2(0x062Bf725dC4cDF947aa79Ca2aaCCD4F385b13b5c);
```

Comment:
I would think you'd want this to be a parameter to support testing on other chains or L2s eventually.

Reply:
maybe just override this in case of testing on another chain, I think it's pretty easy to do

Reply:
It's something Alchemix can change easily enough in any case.